apps/overige-objecten-api/src/__mocks__/objectsResponseData.ts

```diff
@@ -70,7 +70,7 @@ export const objectsResponseData = ({ type }: { type?: 'kennisartikel' | 'vac' }
             status: 'actief',
             antwoord:
               'U moet een afspraak maken bij de gemeente, uw identiteitsbewijs meenemen en een recente pasfoto aanleveren.',
-            doelgroep: 'eu_burger',
+            doelgroep: 'eu-burger',
```

Comment:
Hoe komt de aanpassing van `_` naar `-`? Is dit een breaking change voor acc/prod?

Reply:
Dit was een bugfix-wijziging, omdat Strapi kebab-case naar snake_case converteert.